pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -42,7 +42,7 @@ type HyperConvergedSpec struct {
 	// featureGates is a map of feature gate flags. Setting a flag to `true` will enable
 	// the feature. Setting `false` or removing the feature gate, disables the feature.
 	// +optional
-	// +kubebuilder:default={}
+	// +kubebuilder:default={withHostModelCPU: true, withHostPassthroughCPU: false}
```

Comment:
do we have to manually maintain this list?

Reply:
Unfortunately, yes

Reply:
ok, please add a TODO in the code to remember us that we should manually maintain it until we'll find how to properly automate this